content/custom-commands/commands.md

```diff
@@ -0,0 +1,314 @@
+
+</center>
+
+## Creating a custom command
+
+Clicking the Create a new Custom Command button (**1**) will instantly create a new command within the selected group (**6**) then redirect you to the edit page for the new CC.
+
+A new custom command has the default response `Edit this to change the output of the custom command {{.CCID}}!` It also assigns the command an ID determined by the number of commands you have created.
+
+{{% notice style="tip" title="Troubleshooting" %}}
+
+If the bot is sending messages such as this in your server, you are likely accidentally triggering CCs with the default response. Check the commands page to find any CCs with empty responses.
+
+{{% /notice %}}
+
+## Command list
+
+The commands page lists the commands (**2**) in the selected group (**6**). They are ordered by [ID](#id-and-name) and display their name (if set), [trigger type](#trigger-types), and trigger (if applicable). You can expand the command by clicking the down arrow, allowing you to preview the full, syntax-highlighted command response.
+
+### Delete command button
+
+Deleting a custom command (**4**) will **permanently** delete the command after confirmation. This cannot be undone.
+
+### Run now
+
+The Run now button (**5**) appears on [Interval trigger](#hourlyminute-interval) commands. It executes the command and resets the next run time based off the current time.
+
+## Command Groups
+
+Command groups allow you to organize your custom commands and apply role and channel restrictions to multiple commands.
+
+The group tabs at the top of the page (**7**) allow you to switch to any of your created groups. The **+** button allows you to create a new group.
+
+### Editing a group
+
+Selecting a group allows you to edit it. Any changes must be saved (**11**) to take effect.
+
+- **Name** (**8**): Name your custom command group (100 characters max).
+- **Delete group** (**9**): Permanently delete the group after confirmation.
+- **Role/Channel restrictions** (**10**): Restrict commands within the group based on roles or channels executed in.
+- **Save group settings** (**11**): Update the group with the new values.
+
+#### Role/Channel restrictions
+
+Restricting a custom command (**10**) prevents it from triggering when not permitted to. A user who tries to run a custom command with the wrong roles/in the wrong channel will not trigger the response or an error.
+
+Whitelisting roles or channels causes them to be required to run the command.
+
+Blacklisted roles or channels will not be permitted to run the command.
+
+{{% notice style="note" %}}
+
+YAGPDB was raised well and honors a "no" when told "no". In other words, blacklisted roles take precedence over whitelisted roles.
+
+This is a relatively common trip-hazard, so take great care when you set up both whitelisted and blacklisted roles, and subsequently give those roles to your members.
```

Comment:
This feels confusingly worded given how you note blacklisted channels
*and* blacklisted roles — I understand that you copied this from
botlabs-gg/yagpdb-docs-v2#13, but we might want to reword this to also
include the mention about roles, or be generic about a blacklist.

As a tangential note, command overrides as seen in core do not have
blacklisted channels, so this rewording does not exactly apply there.